### DISCUSSION TITLE
Database on littlefs - howto delete/correct

### DISCUSSION THREAD

**Discussion by tomschlde** [RaBa64/OpenDTU#13] [Q&A]
After testing on my own inverter i transferred the DTU to a friend. Unfortunately there where values saved from my plant at first, so that there are wrong values displayed from the first day (23,8,25,15,11205.999374) . (dark blue)
<img width="503" alt="image" src="https://github.com/RaBa64/OpenDTU-Database/assets/59179041/ee624f5b-5c8e-4d3a-8d03-c8795bc2d106">
The databes can be downloaded - can is be uploaded again?
How can the database be deleted?

`[[23,8,25,15,11205.999374],[23,8,25,16,11319.999695],[23,8,25,17,11379.000664],[23,8,25,18,11430.999756],[23,8,25,19,11447.000504],[23,8,25,20,11451.000214],[23,8,26,5,11451.999664],[23,8,26,6,11453.000069],[23,8,26,7,11475.999832],[23,8,26,8,11743.000031],[23,8,26,9,12253.999710],[23,8,26,10,12701.999664],[23,8,26,11,13010.999680],[23,8,26,12,13296.000481],[23,8,26,13,13481.999397],[23,8,26,14,13682.000160],[23,8,26,15,13885.000229],[23,8,26,16,14038.999557],[23,8,26,17,14104.000092],[23,8,26,18,14141.000748],[23,8,26,19,14161.999702],[23,8,26,20,14163.999557],[23,8,27,6,14164.999962],[23,8,27,7,14171.999931],[23,8,27,8,14288.000107],[23,8,27,12,1065276.000977],[23,8,27,13,1066040.039062],[23,8,27,14,1066631.958008],[23,8,27,15,1067209.960938],[23,8,27,16,1067554.077148],[23,8,27,17,1068227.905273],[23,8,27,18,1068515.991211],[23,8,27,19,1068587.036133],[23,8,27,20,1068599.975586],[23,8,28,6,1068604.003906],[23,8,28,7,1068634.033203],[23,8,28,8,1068785.888672],[23,8,28,9,1069255.004883],[23,8,28,10,1069854.980469],[23,8,28,11,1070318.969727],[23,8,28,12,1070855.957031],[23,8,28,13,1071484.008789],[23,8,28,14,1072068.969727],[23,8,28,15,1072490.966797],[23,8,28,16,1072802.978516],[23,8,28,17,1073061.035156],[23,8,28,18,1073201.049805],[23,8,28,19,1073260.986328],[23,8,28,20,1073265.991211],[23,8,29,6,1073266.967773],[23,8,29,7,1073302.978516],[23,8,29,8,1073409.057617],[23,8,29,9,1073588.867188],[23,8,29,10,1074044.921875],[23,8,29,11,1074812.988281],[23,8,29,12,1075604.003906],[23,8,29,13,1076338.989258],[23,8,29,14,1077113.037109],[23,8,29,15,1077756.958008],[23,8,29,16,1078356.933594],[23,8,29,17,1078789.916992],[23,8,29,18,1079076.904297],[23,8,29,19,1079120.971680],[23,8,29,20,1079122.924805],[23,8,30,6,1079124.023438],[23,8,30,7,1079151.977539],[23,8,30,8,1079310.058594],[23,8,30,9,1079722.045898],[23,8,30,10,1080165.039062],[23,8,30,11,1080445.068359],[23,8,30,12,1080599.121094],[23,8,30,13,1080778.930664],[23,8,30,14,1080879.028320],[23,8,30,15,1080962.036133],[23,8,30,16,1081197.021484],[23,8,30,17,1081821.044922],[23,8,30,18,1082056.030273],[23,8,30,19,1082161.010742],[23,8,30,20,1082167.968750],[23,8,31,6,1082168.945312],[23,8,31,7,1082200.927734],[23,8,31,8,1082366.943359],[23,8,31,9,1082958.007812],[23,8,31,10,1083764.038086],[23,8,31,11,1084535.034180],[23,8,31,12,1085114.013672],[23,8,31,13,1085426.025391],[23,8,31,14,1085872.924805],[23,8,31,15,1086499.023438],[23,8,31,16,1087050.048828],[23,8,31,17,1087474.975586],[23,8,31,18,1087708.007812],[23,8,31,19,1087769.042969],[23,8,31,20,1087775.024414],[23,9,1,6,1087776.000977],[23,9,1,7,1087800.048828],[23,9,1,8,1087862.060547],[23,9,1,9,1087920.043945],[23,9,1,10,1088016.113281],[23,9,1,11,1088144.042969],[23,9,1,12,1088389.038086],[23,9,1,13,1088512.939453],[23,9,1,14,1088760.986328],[23,9,1,15,1089032.958984],[23,9,1,16,1089266.967773],[23,9,1,17,1089895.019531],[23,9,1,18,1090209.960938],[23,9,1,19,1090279.052734],[23,9,1,20,1090282.958984],[23,9,2,6,1090283.935547],[23,9,2,7,1090335.937500],[23,9,2,8,1090501.953125],[23,9,2,9,1090922.973633],[23,9,2,10,1091661.010742],[23,9,2,11,1092468.017578],[23,9,2,12,1093130.981445],[23,9,2,13,1093854.980469],[23,9,2,14,1094547.119141],[23,9,2,15,1095142.944336],[23,9,2,16,1095640.014648],[23,9,2,17,1096285.034180],[23,9,2,18,1096572.998047],[23,9,2,19,1096638.916016],[23,9,2,20,1096641.967773],[23,9,3,6,1096654.052734],[23,9,3,7,1096687.011719],[23,9,3,8,1096851.074219],[23,9,3,9,1097058.105469],[23,9,3,10,1097467.041016],[23,9,3,11,1098165.039062],[23,9,3,12,1098981.933594],[23,9,3,13,1099770.996094],[23,9,3,14,1100536.987305],[23,9,3,15,1101286.865234],[23,9,3,16,1102003.051758],[23,9,3,17,1102566.040039],[23,9,3,18,1102851.074219],[23,9,3,19,1102935.058594],[23,9,3,20,1102939.086914],[23,9,4,6,1102942.993164],[23,9,4,7,1102976.074219],[23,9,4,8,1103130.004883],[23,9,4,9,1103664.062500],[23,9,4,10,1104459.960938],[23,9,4,11,1105272.949219],[23,9,4,12,1106089.965820],[23,9,4,13,1106905.029297],[23,9,4,14,1107719.970703],[23,9,4,15,1108536.010742],[23,9,4,16,1109351.928711],[23,9,4,17,1110013.061523],[23,9,4,18,1110322.021484],[23,9,4,19,1110385.986328],[23,9,4,20,1110386.962891],[23,9,5,6,1110388.061523],[23,9,5,7,1110407.958984],[23,9,5,8,1110559.082031],[23,9,5,9,1111138.061523],[23,9,5,10,1112020.996094],[23,9,5,11,1113125.000000],[23,9,5,12,1114364.990234],[23,9,5,13,1115661.132812],[23,9,5,14,1116926.025391],[23,9,5,15,1118073.974609],[23,9,5,16,1119020.019531],[23,9,5,17,1119698.974609],[23,9,5,18,1120042.968750],[23,9,5,19,1120107.910156],[23,9,5,20,1120110.961914],[23,9,6,6,1120111.938477],[23,9,6,7,1120131.958008],[23,9,6,8,1120281.005859],[23,9,6,9,1120862.060547],[23,9,6,10,1121751.953125],[23,9,6,11,1122855.957031],[23,9,6,12,1124082.031250],[23,9,6,13,1125367.919922],[23,9,6,14,1126623.046875],[23,9,6,15,1127760.986328],[23,9,6,16,1128704.956055],[23,9,6,17,1129368.041992],[23,9,6,18,1129689.941406],[23,9,6,19,1129750.976562],[23,9,6,20,1129753.051758],[23,9,7,6,1129754.028320],[23,9,7,7,1129778.076172],[23,9,7,8,1129929.931641],[23,9,7,9,1130492.919922],[23,9,7,10,1131356.933594],[23,9,7,11,1132442.993164],[23,9,7,12,1133654.052734],[23,9,7,13,1134924.072266],[23,9,7,14,1136175.048828],[23,9,7,15,1137307.006836],[23,9,7,16,1138243.041992],[23,9,7,17,1138900.024414],[23,9,7,18,1139223.022461],[23,9,7,19,1139280.029297],[23,9,8,6,1139281.005859],[23,9,8,7,1139303.955078],[23,9,8,8,1139449.951172],[23,9,8,9,1140002.929688],[23,9,8,10,1140850.952148],[23,9,8,11,1141910.034180],[23,9,8,12,1143088.012695],[23,9,8,13,1144314.941406],[23,9,8,14,1145516.967773],[23,9,8,15,1146603.027344],[23,9,8,16,1147487.060547],[23,9,8,17,1148096.923828],[23,9,8,18,1148382.080078],[23,9,8,19,1148427.001953],[23,9,8,20,1148427.978516],[23,9,9,6,1148429.077148],[23,9,9,7,1148453.979492],[23,9,9,8,1148604.003906],[23,9,9,9,1149109.008789],[23,9,9,10,1149905.029297],[23,9,9,11,1150919.067383],[23,9,9,12,1152056.030273],[23,9,9,13,1153250.976562],[23,9,9,14,1154438.964844],[23,9,9,15,1155514.038086],[23,9,9,16,1156376.953125],[23,9,9,17,1156968.994141],[23,9,9,18,1157239.990234],[23,9,9,19,1157286.865234],[23,9,9,20,1157287.963867],[23,9,10,6,1157289.062500],[23,9,10,7,1157312.988281],[23,9,10,8,1157458.007812],[23,9,10,9,1157939.941406],[23,9,10,10,1158700.927734],[23,9,10,11,1159675.048828],[23,9,10,12,1160761.962891],[23,9,10,13,1161911.987305],[23,9,10,14,1163032.958984],[23,9,10,15,1163930.908203],[23,9,10,16,1164729.980469],[23,9,10,17,1165225.952148],[23,9,10,18,1165457.031250],[23,9,10,19,1165500.976562],[23,9,10,20,1165501.953125],[23,9,11,6,1165502.929688],[23,9,11,7,1165526.000977],[23,9,11,8,1165671.020508],[23,9,11,9,1166151.000977],[23,9,11,10,1166932.006836],[23,9,11,11,1167920.043945],[23,9,11,12,1169023.925781],[23,9,11,13,1170192.016602],[23,9,11,14,1171322.998047],[23,9,11,15,1172314.941406],[23,9,11,16,1172913.940430],[23,9,11,17,1173333.984375],[23,9,11,18,1173576.049805],[23,9,11,19,1173600.097656],[23,9,12,6,1173601.074219],[23,9,12,7,1173624.023438],[23,9,12,8,1173752.929688],[23,9,12,9,1174145.996094],[23,9,12,10,1174858.032227],[23,9,12,11,1175781.005859],[23,9,12,12,1176763.916016],[23,9,12,13,1177839.965820],[23,9,12,14,1178920.043945],[23,9,12,15,1179998.046875],[23,9,12,16,1180524.047852],[23,9,12,17,1180988.037109],[23,9,12,18,1181029.052734],[23,9,12,19,1181031.005859],[23,9,13,6,1181031.982422],[23,9,13,7,1181038.940430],[23,9,13,8,1181071.044922],[23,9,13,9,1181166.015625],[23,9,13,10,1181593.994141],[23,9,13,11,1182001.953125],[23,9,13,12,1182685.058594],[23,9,13,13,1183105.957031],[23,9,13,14,1183298.095703],[23,9,13,15,1183838.012695],[23,9,13,16,1184473.999023],[23,9,13,17,1184723.022461],[23,9,13,18,1184771.972656],[23,9,13,19,1184787.963867],[23,9,14,6,1184789.062500],[23,9,14,7,1184807.983398],[23,9,14,8,1184943.969727],[23,9,14,9,1185362.060547],[23,9,14,10,1186158.935547],[23,9,14,11,1187247.070312],[23,9,14,12,1187896.972656],[23,9,14,13,1188453.979492],[23,9,14,14,1189095.947266],[23,9,14,15,1189939.941406],[23,9,14,16,1190375.000000],[23,9,14,17,1190925.048828],[23,9,14,18,1191145.996094],[23,9,14,19,1191183.959961],[23,9,15,6,1191184.936523],[23,9,15,7,1191204.956055],[23,9,15,8,1191335.937500],[23,9,15,9,1191895.019531],[23,9,15,10,1192756.958008],[23,9,15,11,1193830.932617],[23,9,15,12,1195036.010742],[23,9,15,13,1196302.978516],[23,9,15,14,1197534.912109],[23,9,15,15,1198638.061523],[23,9,15,16,1199338.989258],[23,9,15,17,1199851.928711],[23,9,15,18,1200009.033203],[23,9,15,19,1200041.015625],[23,9,16,6,1200041.992188],[23,9,16,7,1200068.969727],[23,9,16,8,1200191.894531],[23,9,16,9,1200697.998047],[23,9,16,10,1201510.986328],[23,9,16,11,1202497.070312],[23,9,16,12,1203614.013672],[23,9,16,13,1204733.032227],[23,9,16,14,1205673.950195],[23,9,16,15,1206364.990234],[23,9,16,16,1206641.967773],[23,9,16,17,1206748.046875],[23,9,16,18,1206791.015625],[23,9,16,19,1206800.048828],[23,9,17,6,1206801.025391],[23,9,17,7,1206808.959961],[23,9,17,8,1206942.016602],[23,9,17,9,1207457.031250],[23,9,17,10,1208241.943359],[23,9,17,11,1209264.892578],[23,9,17,12,1210214.965820],[23,9,17,13,1211441.040039],[23,9,17,14,1212487.060547],[23,9,17,15,1212994.995117],[23,9,17,16,1213400.024414],[23,9,17,17,1213688.964844],[23,9,17,18,1213807.006836],[23,9,17,19,1213822.998047],[23,9,18,6,1213823.974609],[23,9,18,7,1213831.054688],[23,9,18,8,1213901.977539],[23,9,18,9,1214034.057617],[23,9,18,10,1214270.019531],[23,9,18,11,1214677.978516],[23,9,18,12,1215745.971680],[23,9,18,13,1216782.958984],[23,9,18,14,1217666.992188],[23,9,18,15,1218750.000000],[23,9,18,16,1219585.937500],[23,9,18,17,1220159.912109],[23,9,18,18,1220390.991211],[23,9,18,19,1220395.996094],[23,9,19,6,1220396.972656],[23,9,19,7,1220411.987305],[23,9,19,8,1220528.930664],[23,9,19,9,1220858.032227],[23,9,19,10,1221437.011719],[23,9,19,11,1222279.052734],[23,9,19,12,1223126.953125],[23,9,19,13,1224136.962891],[23,9,19,14,1225297.973633],[23,9,19,15,1226035.034180],[23,9,19,16,1226605.957031],[23,9,19,17,1226833.007812],[23,9,19,18,1226954.956055],[23,9,19,19,1226967.041016],[23,9,20,6,1226968.017578],[23,9,20,7,1226979.003906],[23,9,20,8,1227064.941406],[23,9,20,9,1227246.093750],[23,9,20,10,1227727.050781],[23,9,20,11,1228340.942383],[23,9,20,12,1229022.949219],[23,9,20,13,1229697.998047],[23,9,20,14,1230499.023438],[23,9,20,15,1231421.020508],[23,9,20,16,1232167.968750],[23,9,20,17,1232729.980469],[23,9,20,18,1232856.933594],[23,9,20,19,1232870.971680],[23,9,21,6,1232871.948242],[23,9,21,7,1232899.047852],[23,9,21,8,1233037.963867],[23,9,21,9,1233303.955078],[23,9,21,10,1233569.946289],[23,9,21,11,1233751.953125],[23,9,21,12,1233884.033203],[23,9,21,13,1234011.962891],[23,9,21,14,1234142.089844],[23,9,21,15,1234225.097656],[23,9,21,16,1234262.084961],[23,9,21,17,1234319.946289],[23,9,21,18,1234344.970703],[23,9,21,19,1234346.069336],[23,9,22,6,1234347.045898],[23,9,22,7,1234354.003906],[23,9,22,8,1234486.083984],[23,9,22,9,1234896.972656],[23,9,22,10,1235603.027344],[23,9,22,11,1236574.951172],[23,9,22,12,1237776.123047],[23,9,22,13,1238847.900391],[23,9,22,14,1239765.991211],[23,9,22,15,1240112.060547],[23,9,22,16,1240208.984375],[23,9,22,17,1240404.052734],[23,9,22,18,1240666.992188],[23,9,22,19,1240676.025391],[23,9,23,6,1240677.001953],[23,9,23,7,1240687.011719],[23,9,23,8,1240828.979492],[23,9,23,9,1241271.972656],[23,9,23,10,1241698.974609],[23,9,23,11,1242482.910156],[23,9,23,12,1243244.018555],[23,9,23,13,1243842.041016],[23,9,23,14,1244454.101562],[23,9,23,15,1245363.037109],[23,9,23,16,1245859.008789],[23,9,23,17,1245948.974609],[23,9,23,18,1245970.947266],[23,9,23,19,1245975.952148],[23,9,24,6,1245977.050781],[23,9,24,7,1245987.060547],[23,9,24,8,1246130.859375],[23,9,24,9,1246585.937500],[23,9,24,10,1247333.007812],[23,9,24,11,1248399.047852],[23,9,24,12,1249572.021484],[23,9,24,13,1250849.975586],[23,9,24,14,1252101.074219],[23,9,24,15,1253212.036133],[23,9,24,16,1254098.999023],[23,9,24,17,1254678.955078],[23,9,24,18,1254889.892578],[23,9,24,19,1254896.972656],[23,9,25,6,1254897.949219],[23,9,25,7,1254909.912109],[23,9,25,8,1255066.894531],[23,9,25,9,1255403.076172],[23,9,25,10,1256007.934570],[23,9,25,11,1256954.956055],[23,9,25,12,1258035.888672],[23,9,25,13,1259134.033203],[23,9,25,14,1260164.062500],[23,9,25,15,1261140.991211],[23,9,25,16,1261776.977539],[23,9,25,17,1262355.957031],[23,9,25,18,1262543.945312],[23,9,25,19,1262549.072266],[23,9,26,6,1262550.048828],[23,9,26,7,1262560.913086],[23,9,26,8,1262690.063477],[23,9,26,9,1263094.116211],[23,9,26,10,1263776.977539],[23,9,26,11,1264741.943359],[23,9,26,12,1265854.003906],[23,9,26,13,1267029.052734],[23,9,26,14,1268166.015625],[23,9,26,14,1268583.007812],[23,9,26,15,1269114.013672],[23,9,26,16,1269522.949219],[23,9,26,17,1269713.134766],[23,9,26,18,1269781.982422],[23,9,26,19,1269787.109375],[23,9,27,6,1269788.085938],[23,9,27,7,1269802.978516],[23,9,27,8,1269885.009766],[23,9,27,9,1270136.962891],[23,9,27,10,1270779.052734],[23,9,27,11,1271701.904297],[23,9,27,12,1272761.962891],[23,9,27,13,1273739.990234],[23,9,27,14,1274804.931641],[23,9,27,15,1275539.062500],[23,9,27,16,1276200.927734],[23,9,27,17,1276541.992188],[23,9,27,18,1276588.989258],[23,9,27,19,1276593.017578],[23,9,28,6,1276593.994141],[23,9,28,7,1276601.074219],[23,9,28,8,1276711.914062],[23,9,28,9,1276987.915039],[23,9,28,10,1277576.049805],[23,9,28,11,1277945.068359],[23,9,28,12,1278570.068359],[23,9,28,13,1279446.044922],[23,9,28,14,1280218.017578],[23,9,28,15,1280908.935547],[23,9,28,16,1281413.940430],[23,9,28,17,1281619.018555],[23,9,28,18,1281662.963867],[23,9,28,19,1281666.992188],[23,9,29,6,1281667.968750],[23,9,29,7,1281677.001953],[23,9,29,8,1281738.037109],[23,9,29,9,1281887.939453],[23,9,29,10,1282356.933594],[23,9,29,11,1282568.969727],[23,9,29,12,1283234.985352],[23,9,29,13,1283827.026367],[23,9,29,14,1284145.019531],[23,9,29,15,1284621.093750],[23,9,29,16,1284941.040039],[23,9,29,17,1284984.008789],[23,9,29,18,1284996.948242],[23,9,29,19,1285000.000000],[23,9,30,6,1285000.976562],[23,9,30,7,1285004.028320],[23,9,30,8,1285122.924805],[23,9,30,9,1285405.029297],[23,9,30,10,1286083.007812],[23,9,30,11,1287033.935547],[23,9,30,12,1288143.920898],[23,9,30,13,1288654.052734],[23,9,30,14,1289227.050781],[23,9,30,15,1289942.993164],[23,9,30,16,1290274.047852],[23,9,30,17,1290478.027344],[23,9,30,18,1290562.011719],[23,9,30,19,1290568.969727],[23,10,1,6,1290569.946289],[23,10,1,7,1290572.021484],[23,10,1,8,1290676.025391],[23,10,1,9,1290935.058594],[23,10,1,10,1291597.900391],[23,10,1,11,1292567.016602],[23,10,1,12,1293596.923828],[23,10,1,13,1294732.055664],[23,10,1,14,1295865.966797],[23,10,1,15,1296867.065430],[23,10,1,16,1297644.042969],[23,10,1,17,1298105.102539],[23,10,1,18,1298265.014648],[23,10,1,19,1298270.996094],[23,10,2,6,1298271.972656],[23,10,2,7,1298275.024414],[23,10,2,8,1298367.065430],[23,10,2,9,1298590.087891],[23,10,2,10,1299178.955078],[23,10,2,11,1300063.964844],[23,10,2,12,1301046.997070],[23,10,2,13,1302118.041992],[23,10,2,14,1303038.085938],[23,10,2,15,1303796.997070],[23,10,2,16,1304301.025391],[23,10,2,17,1304669.067383],[23,10,2,18,1304786.010742],[23,10,2,19,1304788.085938],[23,10,3,7,1304789.062500],[23,10,3,8,1304807.983398],[23,10,3,9,1304899.047852],[23,10,3,10,1305086.914062],[23,10,3,11,1305288.940430],[23,10,3,12,1305407.958984],[23,10,3,13,1305503.906250],[23,10,3,14,1305691.894531],[23,10,3,15,1306113.037109],[23,10,3,16,1306560.058594],[23,10,3,17,1306765.014648],[23,10,3,18,1306843.017578],[23,10,3,19,1306844.970703],[23,10,4,6,1306845.947266],[23,10,4,7,1306848.999023],[23,10,4,8,1306947.998047],[23,10,4,9,1307181.030273],[23,10,4,10,1307800.048828],[23,10,4,11,1308570.922852],[23,10,4,12,1309292.968750],[23,10,4,13,1309723.876953],[23,10,4,14,1310217.041016],[23,10,4,15,1310495.971680],[23,10,4,16,1310698.974609],[23,10,4,17,1310901.000977],[23,10,4,18,1310958.984375],[23,10,4,19,1310960.937500],[23,10,5,7,1310961.914062],[23,10,5,8,1311005.004883],[23,10,5,9,1311127.929688],[23,10,5,10,1311396.972656],[23,10,5,11,1311680.053711],[23,10,5,12,1311898.925781],[23,10,5,13,1312412.963867],[23,10,5,14,1313281.005859],[23,10,5,15,1313729.003906],[23,10,5,16,1314114.013672],[23,10,5,17,1314530.029297],[23,10,5,18,1314626.953125],[23,10,5,19,1314627.929688],[23,10,6,6,1314629.028320],[23,10,6,7,1314631.958008],[23,10,6,8,1314745.971680],[23,10,6,9,1315002.929688],[23,10,6,10,1315553.955078],[23,10,6,11,1316438.964844],[23,10,6,12,1317430.908203],[23,10,6,13,1318520.996094],[23,10,6,14,1319585.083008],[23,10,6,15,1320520.996094],[23,10,6,16,1321204.101562],[23,10,6,17,1321526.000977],[23,10,6,18,1321612.060547],[23,10,6,19,1321613.037109],[23,10,7,7,1321614.013672],[23,10,7,8,1321647.949219],[23,10,7,9,1321762.939453],[23,10,7,10,1321963.989258],[23,10,7,11,1322655.029297],[23,10,7,12,1323648.925781],[23,10,7,13,1324684.936523],[23,10,7,14,1325526.000977],[23,10,7,15,1326332.031250],[23,10,7,16,1327006.958008],[23,10,7,17,1327370.971680],[23,10,7,18,1327433.959961],[23,10,8,7,1327434.936523],[23,10,8,8,1327471.069336],[23,10,8,9,1327583.984375],[23,10,8,10,1327993.041992],[23,10,8,11,1328535.888672],[23,10,8,12,1329279.052734],[23,10,8,13,1330048.095703],[23,10,8,14,1330383.911133],[23,10,8,15,1330714.111328],[23,10,8,16,1330988.037109],[23,10,8,17,1331186.035156],[23,10,8,18,1331213.989258],[23,10,9,7,1331214.965820],[23,10,9,8,1331238.037109],[23,10,9,9,1331314.941406],[23,10,9,10,1331483.032227],[23,10,9,11,1331867.919922],[23,10,9,12,1332296.997070],[23,10,9,13,1332708.007812],[23,10,9,14,1333072.021484],[23,10,9,15,1333357.055664],[23,10,9,16,1333744.018555],[23,10,9,17,1333942.138672],[23,10,9,18,1333973.999023],[23,10,10,7,1333974.975586],[23,10,10,8,1334036.010742],[23,10,10,9,1334234.008789],[23,10,10,10,1334698.974609],[23,10,10,11,1335248.046875],[23,10,10,12,1336032.958984],[23,10,10,13,1337015.869141],[23,10,10,14,1338071.044922],[23,10,10,15,1338984.008789],[23,10,10,16,1339655.029297],[23,10,10,17,1339973.022461],[23,10,10,18,1340034.057617],[23,10,11,7,1340035.034180],[23,10,11,8,1340104.980469],[23,10,11,9,1340270.019531],[23,10,11,10,1340733.886719],[23,10,11,11,1341441.894531],[23,10,11,12,1342335.937500],[23,10,11,13,1343234.008789],[23,10,11,14,1344207.031250],[23,10,11,15,1344869.018555],[23,10,11,16,1345183.959961],[23,10,11,17,1345402.954102],[23,10,11,18,1345428.100586],[23,10,12,7,1345429.077148],[23,10,12,8,1345436.035156],[23,10,12,9,1345459.960938],[23,10,12,10,1345526.977539],[23,10,12,11,1345729.003906],[23,10,12,12,1346063.964844],[23,10,12,13,1346417.968750],[23,10,12,14,1346538.085938],[23,10,12,15,1346651.977539],[23,10,12,16,1346737.060547],[23,10,12,17,1346783.081055],[23,10,12,18,1346791.992188],[23,10,13,7,1346792.968750],[23,10,13,8,1346811.889648],[23,10,13,9,1346895.019531],[23,10,13,10,1347140.991211],[23,10,13,11,1347744.140625],[23,10,13,12,1348614.013672],[23,10,13,13,1348850.097656],[23,10,13,14,1349016.967773],[23,10,13,15,1349135.986328],[23,10,13,16,1349185.058594],[23,10,13,17,1349208.984375],[23,10,13,18,1349212.890625],[23,10,14,7,1349213.989258],[23,10,14,8,1349281.005859],[23,10,14,9,1349414.062500],[23,10,14,10,1349888.061523],[23,10,14,11,1350523.925781],[23,10,14,12,1351174.072266],[23,10,14,13,1351783.935547],[23,10,14,14,1352480.957031],[23,10,14,15,1352835.083008],[23,10,14,16,1353171.997070],[23,10,14,17,1353225.952148],[23,10,14,18,1353238.037109],[23,10,15,7,1353239.013672],[23,10,15,8,1353255.004883],[23,10,15,9,1353413.085938],[23,10,15,10,1353515.991211],[23,10,15,11,1353731.079102]]`

**Comment by RaBa64** [comment]
You could erase the flash and use the instructions to install OpenDTU on a new device.

**Reply by MartinNK** [reply]
Any chance we get a restore database function? I tried to get the code manually an check how it could be done, but I am an old C, Assembler, html, JS, ... programmer not happy with all these C++ and C# 😪

**Reply by RaBa64** [reply]
https://github.com/RaBa64/OpenDTU-Database/releases/tag/v24.2.18-Database

**Reply by RaBa64** [reply]
To delete the database, you can create a 0 bytes files and restore it.

**Reply by MartinNK** [reply]
Thanks ... now it works with 24.2.24 ... some small problems with restarting graphic after 2 days missing data wahrscheinlich wegen alter version 24.2.18 und fehler ...

![Screenshot 2024-02-26 at 15-20-25 OpenDTU](https://github.com/RaBa64/OpenDTU-Database/assets/3524916/367861a7-a7bf-4d13-b2e7-5a28dc4a3c57)

Just copied some database files together to get all data again since first backup of database.bin ... works fine

Linux command to copy all backups together ... 
  cat database0.bin database1.bin > database.bin
